src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-snapshotschedule-list/cephfs-snapshotschedule-list.component.html

```diff
@@ -4,6 +4,18 @@
   </cd-loading-panel>
 </ng-container>
 
+<cd-alert-panel *ngIf="!(snapScheduleModuleStatus$ | async)"
+                type="info"
+                spacingClass="mb-3"
+                i18n>
+In order to access the snapshot scheduler feature, the snap_scheduler module must be enabled
+  <a
+  class="text-decoration-underline"
+  (click)="enableSnapshotSchedule()">
+    Enable the Snapshot Scheduler Module</a
```

Comment:
One thought I have is, rather than having a link why not have a button for this? I mean `Enable` Button rather than link that says `Enable the Snapshot Scheduler Module`. Thoughts? 

https://github.com/ceph/ceph/pull/54750#issuecomment-1840545734

Reply:
I've implemented this way to ensure consistency because that's how it is implemented on RGW. However, I do agree with you that semantically this should be a button. Should I create a tracker to change to a button on RGW as well? Or should I just include it on this PR given that it is such small work?

Reply:
I think we can include it in this PR as this is a small change.

Reply:
done